setup.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+from setuptools import setup
+
+INSTALL_REQUIRES = ["alembic", "click>=7.0"]
+EXTRAS_REQUIRE = {
```

Comment:
two different requires for test and lint look overkill, is this common practice? I thought that people simply do requirements.dev, because it doesn't make much sense to install dev dependencies from pypi. (or you know, you can use poetry 💀 )

Reply:
ha, never mind, you join them together below